x-pack/legacy/plugins/upgrade_assistant/server/lib/reindexing/reindex_actions.test.ts

```diff
@@ -299,9 +299,7 @@ describe('ReindexActions', () => {
   });
 
   describe('runWhileConsumerLocked', () => {
-    Object.keys(IndexGroup).forEach(typeKey => {
-      const consumerType = IndexGroup[typeKey as any] as IndexGroup;
-
+    Object.entries(IndexGroup).forEach(([typeKey, consumerType]) => {
```

Comment:
:information_source: The `as any` typing complained, that `any` is not a valid key to the `IndexGroup` enum. So I actually just used the `entries` method here, which a/ looks nicer and b/ is perfectly type safe now.